client/reader/recommendations/posts/index.jsx

```diff
@@ -0,0 +1,26 @@
+var React = require( 'react' );
```

Comment:
so `reader/recommendations/posts-recommendations`? that seems redundant. maybe we should update the naming guidelines to take subfolders into account?